experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -110,8 +114,33 @@ export const getAbsoluteUrl = (
   ) {
     host += `:${port}`;
   }
+  // Redact sensitive query parameters
+  if (path.includes('?')) {
+    const [pathname, query] = path.split('?', 2);
+    const searchParams = new URLSearchParams(query);
+    let sensitiveParamsToRedact: string[];
+
+    if (redactedQueryParams === undefined) {
+      sensitiveParamsToRedact = Array.from(DEFAULT_QUERY_STRINGS_TO_REDACT); // If redactedQueryParams is not provided, use default redacted query params
+    } else if (redactedQueryParams.length === 0) {
+      sensitiveParamsToRedact = []; // If redactedQueryParams is empty, do not redact any query string params
+    } else {
+      sensitiveParamsToRedact = redactedQueryParams;
+    }
```

Comment:
I think these do not need to be separate cases

```suggestion
    } else {
      sensitiveParamsToRedact = redactedQueryParams;
    }
```